ansible/roles/search_head/files/indexes.conf

```diff
@@ -0,0 +1,123 @@
+[default]
```

Comment:
we should problably set max data size to make sure we don't overload the disk we are creating for the instance, what do you think @P4T12ICK ?

Reply:
Yes I can introduce it.